main.go

```diff
@@ -102,6 +125,8 @@ func (cfg config) getBuilds(f filter) (builds, error) {
 
 type filter struct {
 	Before           int    `url:"before,omitempty"`
+	After            int    `url:"after,omitempty"`
+	Limit            int    `url:"limit,omitempty"`
```

Comment:
added new field but not used anywhere

Reply:
It may have an a different meaning if specified (and empty by default)